src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -211,36 +263,67 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('meta_keyword', TranslatableType::class, [
+                'label' => $this->trans('Meta keywords', 'Admin.Global'),
+                'help' => $this->trans('To add tags, click in the field, write something, and then press the "Enter" key.', 'Admin.Shopparameters.Help')
```

Comment:
```suggestion
                'help' => $this->trans('To add tags, click in the field, write something, and then press the "Enter" key.', 'Admin.Shopparameters.Help')
```

I think this help text needs to be removed entirely. We could add a tooltip that explains Meta keywords next to it, e.g.: _"Search engines use meta keywords to index pages. Only you can see them."_

Ping @PrestaShop/ux-team

Reply:
Hi @l-delin!

It's a great idea to add a tooltip that explains meta keywords but I wouldn't remove the help text, as we have specific behavior here. The merchant can put a space in the tag (this is not always the case). So maybe the user won't understand that to validate his tag he has to press "enter" key and not "space" key. But we can cut the sentence with : "To add tags, press the 'enter' key. Invalid characters: <>;=#{}

Wdyt?

Reply:
Hello @ElodieOS 🙌

How about: "To add tags, press the 'enter' key. You can also use the 'comma' key. Invalid characters: <>;=#{}" ?

Reply:
Hello @l-delin ! 
Yes perfect 👌